.github/CODEOWNERS

```diff
@@ -3,8 +3,8 @@
 # and another the rest of the directory.
 
 # All your base
-*                   @DataDog/container-integrations @Datadog/container-ecosystems
+*                  @Datadog/container-ecosystems
 
 # Documentation
-README.md             @DataDog/documentation @DataDog/container-integrations @Datadog/container-ecosystems
-/docs/                @DataDog/documentation @DataDog/container-integrations @Datadog/container-ecosystems
+README.md             @DataDog/documentation @Datadog/container-ecosystems
```

Comment:
I wonder why it doesn't like this; maybe it needs to be `@Datadog` (lower case "d" in "dog")?

Reply:
Or perhaps the error is actually meant for `@Datadog/container-ecosystems` -> `@DataDog/container-ecosystems`